distro/otel_test.go

```diff
@@ -746,6 +746,24 @@ func assertResource(t *testing.T, attrs []*comm.KeyValue) {
 		},
 	}, "should have proper splunk.distro.version value")
 
+	assert.Contains(t, attrs, &comm.KeyValue{
+		Key: "telemetry.distro.version",
+		Value: &comm.AnyValue{
+			Value: &comm.AnyValue_StringValue{
+				StringValue: distro.Version(),
+			},
+		},
+	}, "should have proper telemetry.distro.version value")
+
+	assert.Contains(t, attrs, &comm.KeyValue{
+		Key: "telemetry.distro.name",
+		Value: &comm.AnyValue{
+			Value: &comm.AnyValue_StringValue{
+				StringValue: distro.Name(),
+			},
+		},
+	}, "should have proper telemetry.distro.name value")
```

Comment:
Can we reorder so that we have first name and then version (like in https://github.com/open-telemetry/semantic-conventions/blob/main/docs/resource/README.md#telemetry-distro)?